### ISSUE TITLE
false "method already declared" error on methodless enums

### ISSUE THREAD

**Issue by EinScott** [beefytech/Beef#1245]
Even though nothing (really) is emitted, that causes an error on ColliderType about methods.

```
[AttributeUsage(.Struct|.Enum)]
struct StuffAttribute : Attribute, IComptimeTypeApply
{
	[Comptime]
	public void ApplyToType(Type type)
	{
		// @report when typing that string, there were lots of crashes
			
		Compiler.EmitTypeBody(type, """
			//
			""");
	}
}

[Stuff]
enum ColliderType
{
	Rect,
	Grid
}
```

**Comment by bfiete** [comment]
Fixed at ed80a8d88bd28becd5532bc3a43b7e090f025a1d
I didn't reproduce the crash, but I believe I fixed what probably caused it.

**Comment by EinScott** [comment]
Yeah you fixed the error, thanks.
I kind of forgot about the \@report comment.. i still have to properly report that.
